EIPS/eip-7642.md

```diff
@@ -0,0 +1,105 @@
+---
+eip: 7642
+title: eth/69 - Drop pre-merge fields
+description: Drop unnecessary fields after the merge
+author: Marius van der Wijden (@MariusVanDerWijden)
+discussions-to: https://ethereum-magicians.org/t/eth-70-drop-pre-merge-fields-from-eth-protocol/19005
+status: Draft
+type: Standards Track
+category: Networking
+created: 2024-02-29
+requires: 5793
+---
+
```

Comment:
```suggestion
```

Reply:
addressed in https://github.com/ethereum/EIPs/pull/8271/commits/e719980ae2d17c3c847de14b542b0ad5e4fe7729 thanks!

Reply:
have a blessed day!